scripts/install-gcs-emu.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+#
+# The MIT License (MIT)
+#
+# Copyright (c) 2019-2020 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+# Installs dependencies for gcs.
+
+die() {
+  echo "$@" 1>&2 ; popd 2>/dev/null; exit 1
+}
+
+install_gcs(){
+    sudo git clone https://github.com/googleapis/google-cloud-cpp/ --branch v1.22.0 /tmp/google-cloud-cpp/
```

Comment:
- I don't think we need `sudo` here.
- Could you try the following to only download the `google/cloud/storage/emulator/` directory?

```
install_gcs(){
    git clone \
      --branch v1.22.0 \
      --depth 1 \
      --filter=blob:none \
      --no-checkout \
      https://github.com/googleapis/google-cloud-cpp/ \
      /tmp/google-cloud-cpp/
    pushd /tmp/google-cloud-cpp/
    git checkout v1.22.0 -- google/cloud/storage/emulator
    popd
    sudo pip3 install -r /tmp/google-cloud-cpp/google/cloud/storage/emulator/requirements.txt
}
```

Reply:
Instead of git, can we use wget/curl to download the release archive? That will avoid the need for git installed and reduce from having to pull in all the git history too.